ugs-core/src/com/willwinder/universalgcodesender/services/JogService.java

```diff
@@ -30,19 +31,11 @@ This file is part of Universal Gcode Sender (UGS).
  */
 public class JogService {
     private static final Logger logger = Logger.getLogger(JogService.class.getSimpleName());
```

Comment:
Got wierd errors because of these state fields never got updated. Now uses the settings instead.